.github/workflows/ci-pipeline.yml

```
@@ -0,0 +1,128 @@
+    steps:
+      - uses: 8398a7/action-slack@v3
+        with:
+          status: ${{ job.status }}
+          fields: repo,message,commit,author,action,eventName,ref,workflow,job,took
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+  # 7. SECURITY: Debricked Scan (Conditional)
+  vulnerabilities-scan:
+    runs-on: ubuntu-latest
+    if: ${{ secrets.DEBRICKED_TOKEN != '' }}
```

actionlint (1.7.10)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)